mlflow/resource_model.go

```diff
@@ -0,0 +1,126 @@
+package mlflow
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// Model defines the parameters that can be set in the resource.
+type Model struct {
+	Name        string `json:"name" tf:"force_new"`
+	Tags        []Tag  `json:"tags,omitempty" tf:"force_new"`
```

Comment:
is it really forcing new?

Reply:
yes, you can't update model names, if it's required a new name, then it is a new object for the API.